libvast/src/detail/posix.cpp

```diff
@@ -137,7 +137,7 @@ caf::error uds_datagram_sender::send(std::span<char> data, int timeout_usec) {
   // the happy path.
   if (::sendto(src_fd, data.data(), data.size(), 0,
                reinterpret_cast<sockaddr*>(&dst), sizeof(struct sockaddr_un))
-      == 0)
+      >= 0)
```

Comment:
If the return value is the number of bytes sent, shouldn't we actually check whether it's exactly `data.size()` here and below?

Reply:
Fair point. I'm afraid we have to come up with a new error code for that though.